src/v3/src/util/generateUUID.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+// eslint-disable-next-line max-len
+export const generateUUID = (): string => Date.now().toString(36)
+  + Math.random().toString(36).substring(2);
```

Comment:
is a UUID explicitly necessary? If so, then we should use a real UUID generator library. If we only need a random string, then we should rename this.